homeassistant/components/mastodon/__init__.py

```diff
@@ -80,6 +81,39 @@ async def async_unload_entry(hass: HomeAssistant, entry: MastodonConfigEntry) ->
     )
 
 
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old config."""
+
+    if entry.version == 1 and entry.minor_version == 1:
+        # Version 1.1 had the unique_id as client_id, this isn't necessarily unique
+        LOGGER.debug("Migrating config entry from version %s", entry.version)
+
+        try:
+            _, instance, account = await hass.async_add_executor_job(
+                setup_mastodon,
+                entry,
+            )
+        except MastodonError as ex:
+            LOGGER.error("Migration failed with error %s", ex)
+            return False
+
+        entry.minor_version = 2
+
+        hass.config_entries.async_update_entry(
+            entry,
+            unique_id=slugify(construct_mastodon_username(instance, account)),
+        )
+
+        LOGGER.info(
```

Comment:
Do we usually log this on info?

Reply:
Majority do, some just debug but nothing higher

Reply:
I'd lower this to debug.